packages/components/src/angle-picker-control/index.js

```diff
@@ -39,6 +39,7 @@ const AngleCircle = ( { value, onChange, ...props } ) => {
 
 	const changeAngleToPosition = ( event ) => {
 		const { x: centerX, y: centerY } = angleCircleCenter.current;
+		event.preventDefault();
```

Comment:
It's unclear what this is accomplishing on its own. Would a comment be helpful?

Reply:
@mtias Good idea :). Will add that now